spec/bourbon/functions/assign_inputs_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "spec_helper"
+
+describe "assign-inputs" do
+  before(:all) do
+    ParserSupport.parse_file("functions/assign-inputs")
+    @text_inputs_list = [
```

Comment:
Since this variable is dup'ed in most of the tests below, and since we already have some indirection when pulling in the parsed fixture css, I wonder if we could do with a text-based parser so that we can have the CSS definitions that are under spec side-by-side within the test file.

That being said, I don't think it's worth holding up the PR from being merged. What do you think about opening an issue?